app/components/admin/custom_fields/hierarchy/item_form_component.rb

```diff
@@ -57,6 +57,18 @@ def url
           end
         end
 
+        def input_format
```

Comment:
:yellow_circle: I'd suggest double thinking about the name of this method.

The `ItemFormComponent` has two inputs:

* The one for the label (the big one)
* The one for the score or the short (the small one)

This format here only defines the small one, not the big one.

Brainstorming:

* extra_input_format
* second_input_format

Reply:
chose `secondary_field_format`